runbotics/runbotics-desktop/src/core/websocket/listeners/process.listener.ts

```diff
@@ -96,10 +96,15 @@ export class ProcessListener {
                     `/api/scheduler/tenants/${tenantId}/processes/${processId}`
                 )
                 .then((response) => {
-                    this.logger.log('Starting process: ' + response.data.name);
                     return response.data;
                 });
 
+            if (!process) {
+                throw new Error(`Process not found in tenant bot is authenticated to. Process id: ${processId}, bot tenant id: ${tenantId}`);
```

Comment:
The process was not found in the tenant to which the bot is authenticated. 
is more clear for me